server/cmwell-batch/src/pack/logback.xml

```diff
@@ -0,0 +1,122 @@
+<configuration debug="false">
+    <shutdownHook class="ch.qos.logback.core.hook.DelayingShutdownHook">
+        <delay>5000</delay>

```

Comment:
Add comment to let the reader know this property WILL DELAY PROCESS SHUTDOWN IN 5 SECONDS.
also, let's verify that HC kills rouge processes in a way that allows the process to exit gracefully within those 5 seconds, before it sends SIGKILL